osu.Game/Overlays/Comments/ReplyCommentEditor.cs

```diff
@@ -38,7 +38,8 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            GetContainingInputManager().ChangeFocus(TextBox);
+            if (!TextBox.ReadOnly)
+                GetContainingInputManager().ChangeFocus(TextBox);
```

Comment:
`ChangeFocus(TextBox)` adds a text cursor even when `TextBox.ReadOnly == true`. I'm not sure whether this is a framework-side bug and needs to be changed, but I've worked around it here.

Reply:
It's _prooobably_ a framework bug. Instinctual reaction says that text box shouldn't [unconditionally accept focus](https://github.com/ppy/osu-framework/blob/147424e287a409a6667e413aad0a822fb4aae886/osu.Framework/Graphics/UserInterface/TextBox.cs#L1261) but the impact of changing that to something like `!IsReadOnly && !Disabled.Value` would have to be tested.